52.md

```diff
@@ -178,17 +178,25 @@ This NIP is intentionally not defining who is authorized to attend a calendar ev
 
 This NIP is also intentionally not defining what happens if a calendar event changes after an RSVP is submitted.
 
+The RSVP MUST tag the calendar event it is in response to with an `e` tag on the calendar event's id.
+
+The RSVP MAY tag use an `a` tag to the calendar event's event coordinates to indicate that the RSVP response could hold trueregardless of any future revisions to the calendar event. If the `a` tag is omitted, the author of the RSVP is explicitly indicating that its response holds true for only that specific revision of the calendar event from the `e` tag.
```

Comment:
I'm actually not sure about this part. Allowing RSVP authors to omit `a` means that it's harder for the calendar event author to pull responses. What do you all think?

Reply:
Resolving this comment as I've updated the PR to reflect the review comments.